Fix lower casing for user's nickname (#14272)
decidim/decidim: RELEASE_NOTES.md

You can read more about this change on PR [#14180](https://github.com/decidim/decidim/pull/14180).

### 3.2. Convert nicknames to lowercase

> As of [#14272](https://github.com/decidim/decidim/pull/14272) we are migrating all the nickanames to lowercase to ensure uniqueness within the same organization, and also fix performance issues which affects large databases having many users.

Reviewer: ```suggestion
As of [#14272](https://github.com/decidim/decidim/pull/14272) we are migrating all the nicknames to lowercase fix performance issues which affects large databases having many participants.
```
